BUILD.bazel

```diff
@@ -93,10 +93,9 @@ cc_test(
     srcs = glob([
         "tests/*.cpp",
         "tests/*.h",
-        "include/sonic/*",
```

Comment:
This is added for supporting coverage test. Pls check again.

Reply:
fixed